test/eventrouter.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This is a test suite for the eventrouter
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+
+function get_es_count() {
+    curl_es "$1" "/_search?size=0&q=$2" | python -c "import sys, json; print json.load(sys.stdin)['hits']['total']"
+}
+
+os::log::info Starting eventrouter test at $( date )
+
+espod=$( get_running_pod es )
+eventcount=$( get_es_count "$espod" 'kubernetes.event.verb:*' )
```

Comment:
This isn't needed - entrypoint.sh will do this - probably should be removed from the other tests as well - but not in this PR